include/circt/Dialect/Verif/Passes.td

```diff
@@ -27,4 +27,19 @@ def VerifyClockedAssertLike : Pass<"verify-clocked-assert-like", "hw::HWModuleOp
   let constructor = "circt::verif::createVerifyClockedAssertLikePass()";
 }
 
+/**
```

Comment:
Nit: Alpha-sort?

I'm not sure where this pass best belongs, but it's not obvious it's in the `Verif` dialect since it presently doesn't touch/use anything from that dialect.  No objection but just a note.

Reply:
It doesn't now, but my goal for this pass is for it to also handle the verif contract replacement stuff, which which will be most of what it does. Also it seemed weird to have something specific to modifying a core representation to make it easier for formal verification to be anywhere other than verif.

Reply:
100% sounds great to me! Thanks for the consideration / care :+1:.